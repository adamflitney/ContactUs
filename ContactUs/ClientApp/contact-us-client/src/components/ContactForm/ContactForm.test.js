import React from 'react';
import renderer from 'react-test-renderer';
import ContactUsForm from '../../components/ContactForm/ContactForm';

describe('ContactUsForm', () => {
    const page = renderer.create(<ContactUsForm />);
    it('renders correctly', () => {
        const tree = page.toJSON();
        expect(tree).toMatchSnapshot();
    });

    
})