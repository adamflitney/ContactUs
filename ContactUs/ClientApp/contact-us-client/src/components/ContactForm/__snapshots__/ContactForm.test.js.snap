// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsForm renders correctly 1`] = `
<form
  className="contact-form"
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <label
      htmlFor="contactFormName"
    >
      Name
    </label>
    <input
      className="form-control"
      id="contactFormName"
      name="name"
      onChange={[Function]}
      placeholder="e.g. Joe Bloggs"
      required={true}
      type="text"
      value=""
    />
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="contactFormEmail"
    >
      Email Address
    </label>
    <input
      className="form-control"
      id="contactFormEmail"
      name="email"
      onChange={[Function]}
      placeholder="e.g. joe@bloggs.com"
      required={true}
      type="email"
      value=""
    />
  </div>
  <div
    className="form-group"
  >
    <label
      htmlFor="contactFormMessage"
    >
      Enter your message
    </label>
    <textarea
      className="form-control"
      id="contactFormMessage"
      name="message"
      onChange={[Function]}
      placeholder="Enter your message here."
      required={true}
      rows="4"
      type="text"
      value=""
    />
  </div>
  <button
    className="btn btn-success"
    id="submit-btn"
    type="submit"
  >
    Submit
  </button>
</form>
`;
